crates/sui-json-rpc-types/src/lib.rs

```diff
@@ -1813,10 +1813,6 @@ impl TryFrom<CertifiedTransaction> for SuiCertifiedTransaction {
     fn try_from(cert: CertifiedTransaction) -> Result<Self, Self::Error> {
         let digest = *cert.digest();
         let (data, sig) = cert.into_data_and_sig();
-        // We should always have a signature here.
-        if sig.signature.sig.is_none() {
-            return Err(anyhow::anyhow!("Certified transaction is not signed"));
-        }
```

Comment:
don think this is the right place to check, now that we have None represented by infinity, the signature should fail verify (instead of checking here)